src/client/markdownStyling.ts

```diff
@@ -0,0 +1,53 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+const styleContent = `
+<style>
+.dummy {
+    /* the first rule doesn't get applied for some reason */
```

Comment:
I think the may because of the `<style>` text here. It should not be needed since you're already setting the text content of a style element